scripts/influxdb.service

```diff
@@ -10,9 +10,10 @@ User=influxdb
 Group=influxdb
 LimitNOFILE=65536
 EnvironmentFile=-/etc/default/influxdb2
-ExecStart=/usr/bin/influxd
+ExecStart=/usr/lib/influxdb/scripts/influxd-systemd-start.sh
 KillMode=control-group
 Restart=on-failure
+Type=forking
```

Comment:
The [docs](https://www.freedesktop.org/software/systemd/man/systemd.service.html#Type=) on this say it's recommended to also set [`PIDFile=`](https://www.freedesktop.org/software/systemd/man/systemd.service.html#PIDFile=) when using `forking`. Could we do that?

Reply:
I tried this, passing it `/run/influxdb/influxd.pid`, but that file did not always show up, so I'm not super sure what's going on there. Do you have insight on how/when/by what component that file is created?

Reply:
To clarify, we could add a section in the wrapper script that acts like `init.sh` which touches a pidfile and then passes that path to the systemd config, but I elected not to do that, as a) I couldn't see what difference it made functionally, and 2) I wanted to alter the startup behavior of influxd via systemd as little as possible from the original.

Reply:
I'd expect something like this to work:
1. We put `PIDFile=/run/influxd.pid` (or something else under `/run/`)
2. We update `influxd-systemd-start.sh` to `echo $! > /run/influxd.pid` on the line after `/usr/bin/influxd &`

Reply:
ok, can do